Adding toString() for Rest and WebSocket response

Fenrur opened the pull request:
Adding toString() for Rest and WebSocket response.
(I deleted the old pr, and I will make PR uponnament feature to avoid having full committed)

oleksiilogvin committed:
FTX - Add order history endpoint

Fenrur committed:
Adding toString() for Rest and WebSocket response

mayankpunetha007 committed:
Add Fills for FTX

mayankpunetha007 committed:
Fix Stupidity

pimpcapital committed:
gradle wrapper upgrade

pimpcapital committed:
gradle wrapper upgrade

pimpcapital committed:
Prepare to release 2.13.3

beijunyi commented:
This is a huge change. May I ask what is the primary use case of this? If you just want to record all the REST traffics, there are easier ways to do this.

Fenrur commented:
This PR is use when we want Debeug the data

pimpcapital committed:
README 2.13.3

beijunyi commented:
I am not very keen on adding toString() methods. One may easily forget to update toString() after adding a new field. As for debugging, logging raw REST request/response shall be sufficient in most cases.

pimpcapital committed:
Prepare to release 2.13.4

pimpcapital committed:
Prepare to release 2.13.5

pimpcapital committed:
Prepare to release 2.13.6-SNAPSHOT

pimpcapital committed:
RateLimitCushion

pimpcapital committed:
Prepare to release 2.14.0

pimpcapital committed:
Prepare to release 2.14.1

pimpcapital committed:
ursa wip

pimpcapital committed:
ursa 1.1.2

pimpcapital committed:
Prepare to release 2.14.2

Commit:
Tidy up in 2.14.3

Commit:
gradle parallel

Commit:
auto LF

Commit:
2.14.4-SNAPSHOT

Commit:
2.14.5-SNAPSHOT

Commit:
Prepare to release 2.15.0

Commit:
Prepare to release 2.15.1-SNAPSHOT

Commit:
Prepare to release 2.15.2-SNAPSHOT

Commit:
2.15.3-SNAPSHOT

Commit:
2.15.4-SNAPSHOT

Commit:
2.15.5-SNAPSHOT

Commit:
2.15.6-SNAPSHOT

Commit:
2.15.7-SNAPSHOT

Commit:
2.15.8-SNAPSHOT

Commit:
2.15.9-SNAPSHOT

Commit:
2.15.10-SNAPSHOT

Commit:
2.15.11-SNAPSHOT

Commit:
2.15.12-SNAPSHOT

Commit:
2.15.13-SNAPSHOT

Commit:
2.15.14-SNAPSHOT

Commit:
2.15.15-SNAPSHOT

Commit:
2.15.16-SNAPSHOT

Commit:
2.15.17-SNAPSHOT

Commit:
2.15.18-SNAPSHOT

Commit:
2.15.18-SNAPSHOT

Commit:
2.15.20-SNAPSHOT

Commit:
2.15.21-SNAPSHOT

Commit:
2.15.22-SNAPSHOT

Commit:
2.15.23-SNAPSHOT

Commit:
minor version bump

pimpcapital committed:
Prepare to release 2.15.24

Commit:
PositionChannel

Commit:
bitmex and bybit position channel

Commit:
deribit UserPositionsChannel

Commit:
deribit UserPositionsChannel

Commit:
2.15.25

Commit:
2.15.26

Commit:
2.15.27

Commit:
2.15.28 deribit

Commit:
2.15.29 deribit user.changes

Commit:
2.15.30 longer bybit websocket timeout

Commit:
2.15.31 GetExecutionTradeHistory

Commit:
2.15.32 LiquidityIndKeys

Commit:
2.15.33

Commit:
2.15.34

Commit:
2.15.35

Commit:
2.15.36

nhut committed:
Add support for FTX spot margin APIs:
* Get lending offers
* Get lending info
* Submit lending offer

nhut committed:
Use String instead of Void as it gave 'does not "opens java.lang"' error.

Commit:
2.15.37

Commit:
2.15.38

Commit:
binance trade list

Commit:
binance trade list

Commit:
binance trade list

Commit:
2.15.42

Commit:
bybitlinear WIP

Commit:
2.16.0 with bybit linear

Commit:
2.16.1 with bybit linear

Commit:
2.16.2 with bybit linear

Commit:
2.16.3

Commit:
2.16.4

Commit:
2.16.5-SNAPSHOT

Commit:
2.16.6-SNAPSHOT

Commit:
2.16.7-SNAPSHOT to fix bybit get position

Commit:
2.16.8-SNAPSHOT

Commit:
2.16.9-SNAPSHOT

Commit:
2.16.10-SNAPSHOT

Commit:
2.16.11-SNAPSHOT

Commit:
2.16.12-SNAPSHOT

Commit:
2.16.13-SNAPSHOT print websocket error on illegal state

Commit:
2.16.14-SNAPSHOT print websocket error on illegal state

Commit:
2.16.15-SNAPSHOT bybit websocket fix

Commit:
2.16.16-SNAPSHOT bybit websocket fix

Commit:
2.16.17-SNAPSHOT bybit linear fix

Commit:
2.16.18-SNAPSHOT bybit linear fix

Commit:
2.16.19-SNAPSHOT bybit linear fix

Commit:
2.16.20-SNAPSHOT kraken keep alive

Commit:
minor fix

Commit:
PendingPing

Commit:
2.16.21-SNAPSHOT

pimpcapital committed:
Prepare to release 2.16.22

pimpcapital committed:
deribit result null

pimpcapital committed:
handle deribit error

Commit:
2.16.24-SNAPSHOT

Commit:
2.16.25-SNAPSHOT

Commit:
minor change

Commit:
minor change

Commit:
optional field deribit

Commit:
2.16.28

Commit:
2.16.29

Commit:
2.16.30

pimpcapital committed:
minor update

Commit:
bitfinex wip

Commit:
no need for CI

Commit:
Apply the same code+structure style to merged FTX lending endpoint

Commit:
crypto.com spot wip

Commit:
okx WIP

Commit:
okx market and public endpoints

Commit:
PostTradeOrder

Commit:
PostTradeCancelOrder

Commit:
PostTradeCancelOrder

Commit:
okx WIP

Commit:
okx account config

Commit:
rest user endpoints

Commit:
fix compile

Commit:
2.17.0-SNAPSHOT with okx

Commit:
Rate limits

Commit:
okx code complete

Commit:
okx code complete

Commit:
okx keys

Commit:
okx  public static final int MAX_DEPTH = 400;

Commit:
minor rename

Commit:
minor rename

Commit:
minor rename

Commit:
GetAssetBalances

Commit:
2.17.8 okx

Commit:
2.17.9 okx

Commit:
okx stupid fix

Commit:
okx stupid fix

Commit:
allow okx ping pong

Commit:
allow okx ping pong

Commit:
bitmex ping pong

pimpcapital committed:
wrong binance delivery websocket rate limit

pimpcapital committed:
Merge with upstream commit #af3ff5d7fb31248c0d28c767d75031d2bda67c7a

pimpcapital committed:
Merge with upstream commit #918184122d67bcecbfabde459572e03d92ddd125

Commit:
Merge with upstream commit #c9d2844c944ba66576d23406f26093f9b8a1a14e

Fenrur committed:
Modularized project

Fenrur committed:
Refactoring to Vertx finished

Fenrur committed:
Refactoring to Vertx finished

Fenrur committed:
edit build.gradle

Fenrur committed:
- Remove javax annotation
- Replace GSON -> Vertx Jackson
- update gradle (artifact, group)
- update README.md

Fenrur committed:
edit README.md

Fenrur committed:
edit README.md

Fenrur committed:
remove atomic/volatile operation

Fenrur committed:
Move to new switch expression

Fenrur committed:
Move to new switch expression

Fenrur committed:
edit rootProject name
